src/cargo/util/process_builder.rs

```diff
@@ -129,11 +131,11 @@ impl ProcessBuilder {
     }
 }
 
-pub fn process<T: AsRef<OsStr>>(cmd: T) -> CargoResult<ProcessBuilder> {
+pub fn process<T: AsRef<OsStr>, U: AsRef<OsStr>>(cmd: T, cwd: U) -> CargoResult<ProcessBuilder> {
```

Comment:
Okay, I'll change it then. Just for clarification, should a cwd of `None` panic or look it up in the env?